### PR TITLE
Add Textdistance

### PR DESCRIPTION
**Pull request by cu8blank**
I would like to add the textdistance Python library into conda-forge.

### CODE REVIEW

File: recipes/textdistance/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    - python
+
+test:
+  imports:
+    - textdistance
+
+about:
+  home: https://github.com/orsinium/textdistance
+  license: GPL-3.0
+  license_family: GPL3
```
Comment (synapticarbors) [inline recipes/textdistance/meta.yaml]: `license_family: GPL`

File: recipes/textdistance/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```
Comment (synapticarbors) [inline recipes/textdistance/meta.yaml]: It looks like the `license_file` is included in the source distribution, so `license_file: LICENSE` should be fine. Also delete the license file that's included in the recipe directory.

File: recipes/textdistance/meta.yaml (hunk 1 above)
Comment (cu8blank) [reply recipes/textdistance/meta.yaml]: I corrected this in the meta.yaml file.

File: recipes/textdistance/meta.yaml (hunk 2 above)
Comment (cu8blank) [reply recipes/textdistance/meta.yaml]: I deleted the license file from the recipe directory and I updated the license_file field. Thank you so much for your feedback and help!!!

File: recipes/textdistance/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```
Comment (mariusvniekerk) [inline recipes/textdistance/meta.yaml]: Could this be noarch?

Comment (cu8blank) [reply recipes/textdistance/meta.yaml]: I added noarch. Thank you!

File: recipes/textdistance/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - textdistance
+
+about:
+  home: https://github.com/orsinium/textdistance
+  license: GPL-3.0
```
Comment (jakirkham) [inline recipes/textdistance/meta.yaml]: Actually appears to be `LGPL`.

File: recipes/textdistance/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+  license_family: GPL
```
Comment (jakirkham) [inline recipes/textdistance/meta.yaml]: Same here.

File: recipes/textdistance/meta.yaml (hunk 4 above)
Comment (cu8blank) [reply recipes/textdistance/meta.yaml]: I updated this to LGPL-3.0. Thank you!

File: recipes/textdistance/meta.yaml (hunk 5 above)
Comment (cu8blank) [reply recipes/textdistance/meta.yaml]: I updated this to LGPL.

**Pull request merged**